Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/data/UserAgendaRepo.kt

```diff
@@ -28,5 +28,9 @@ class UserAgendaRepo private constructor(context: Context) {
         sharedPrefs.edit().putStringSet(sessionIdsKey, savedSessionIds).apply()
     }
 
+    fun getMySessionIds(): HashSet<String> {
```

Comment:
unused?

Reply:
yep...was planning on doing something like getAgendaSessionsWhereIdInSet() but Firebase's query API is super crude and doesn't have this :-P